src/stavenote.ts

```diff
@@ -183,7 +244,7 @@ export class StaveNote extends StemmableNote {
     //
     // Special case 1 :: middle voice rest between two notes
     //
-    if (noteM.isrest && !noteU.isrest && !noteL.isrest) {
+    if (noteM?.isrest && !noteU.isrest && !noteL.isrest) {
```

Comment:
`noteM` must be defined here, so better to add a `if (!noteM) throw...` and remove the `noteM?`

Reply:
done